app/app.go

```diff
@@ -1215,10 +1267,12 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(wasm.ModuleName)
 	paramsKeeper.Subspace(rewardtypes.ModuleName)
 	paramsKeeper.Subspace(triggertypes.ModuleName)
+	paramsKeeper.Subspace(oracletypes.ModuleName)
```

Comment:
Don't need this since the oracle module doesn't use the (deprecated) params module.